### ISSUE TITLE
Support for comments in json files

### ISSUE THREAD

**Issue by immanuel-h** [maptiler/tileserver-gl#977]
It would be nice if tileserver-gl(-light) would allow for comments in json files. Especially with larger style files it sometimes is very helpful to have additional comments detailing specific sections and so on. Maybe [json5](https://www.npmjs.com/package/json5) or [strip-json-comments](https://www.npmjs.com/package/strip-json-comments) could be of help here. 

Cheers :)

**Comment by acalcutt** [comment]
I agree that comments sounds nice. I would support a PR that added them.
